### PR TITLE
utils_libvirt/libvirt_disk: add function to get disk by serial

### PR DESCRIPTION
**Pull request by smitterl**
Use the SERIAL to identify a disk and return its path.

Suggested-by: Chunfu Wen <chwen@redhat.com>
Signed-off-by: Sebastian Mitterle <smitterl@redhat.com>

### CODE REVIEW

**Comment by smitterl** [comment]
@chunfuwen Please help review. As mentioned on https://github.com/avocado-framework/avocado-vt/pull/3811#issuecomment-1850699674 unfortunately we can't set serial on LUNs but its implementation might be useful on other occasions.

**Review by chunfuwen — APPROVED** [review]
lgtm

**Comment by chunfuwen** [comment]
@smitterl ,could you upload one simple test result?

File: virttest/utils_libvirt/libvirt_disk.py [hunk 1]
```diff
@@ -102,6 +102,31 @@ def get_non_root_disk_name(session):
         return name.strip(), mpoint.strip()
 
 
+def get_disk_by_serial(session, serial):
```
Comment (dzhengfy) [inline virttest/utils_libvirt/libvirt_disk.py:105]: We already have same function in utils_disk.py.

`def get_disk_by_serial(serial_str, session=None):`

Comment (smitterl) [reply virttest/utils_libvirt/libvirt_disk.py:105]: Thanks for the pointer. I'll close this one.

**Comment by smitterl** [comment]
Closing this as there is already another implementation with the same API.

**Pull request closed without merging**